commercial measures/SWSV016-01 Weather Sealing/SWSV016-01 Weather Sealing_Ex/cohorts.csv

```diff
@@ -0,0 +1,123 @@
+,Htl&0&cPVVG&Ex&BldgShell,Htl/templates/root.pxt,Mass Heavy,"%= codes.lookup(""Opaque Envelope Requirements | Climate Zone #{climate_zone}"", ""Walls Above Grade | Mass Heavy"", ""Cavity R-Value"")['R-IP']","%= (1/codes.lookup(""Opaque Envelope Requirements | Climate Zone #{climate_zone}"", ""Walls Above Grade | Metal Framed"", ""Maximum U-Factor"")-1.49)['R-IP']","%= (1/codes.lookup(""Opaque Envelope Requirements | Climate Zone #{climate_zone}"", ""Roofs | Metal Buildings"", ""Maximum U-Factor"")-0.78)['R-IP']","%= (codes.lookup(""Fenestration Requirements | Climate Zone #{climate_zone}"", ""Fixed Window"", ""U-Factor""))['U-IP']","%= (codes.lookup(""Fenestration Requirements | Climate Zone #{climate_zone}"", ""Fixed Window"", ""SHGC""))","%= (codes.lookup(""Fenestration Requirements | Climate Zone #{climate_zone}"", ""Fixed Window"", ""VT""))","%= ((1/codes.lookup(""Opaque Envelope Requirements | Climate Zone #{climate_zone}"", ""Floors | Other"", ""Maximum U-Factor""))-4.261)['R-IP']","%= codes.lookup(""Lighting Power Density"", ""office_large"", ""LPD"")['W/ft2']","%= codes.lookup(""Lighting Power Density"", ""kitchen"", ""LPD"")['W/ft2']",,,"%= codes.lookup(""Lighting Power Density"", ""corridor"", ""LPD"")['W/ft2']",,,,"%= codes.lookup(""Lighting Power Density"", ""lobby"", ""LPD"")['W/ft2']","%= codes.lookup(""Lighting Power Density"", ""guestroom"", ""LPD"")['W/ft2']",,,"%= codes.lookup(""Lighting Power Density"", ""dining_fine"", ""LPD"")['W/ft2']",,,,,"%= codes.lookup(""Lighting Power Density"", ""laundry"", ""LPD"")['W/ft2']",,,,,,,,,,,"%= codes.lookup(""Economizer Setpoints"", ""#{climate_zone}"", ""Fixed Drybulb"")['F']"
```

Comment:
Comment out Htl rows in this cohorts file, and copy the cohorts file to the Htl_Ex folder.

Reply:
To clarify:
1. In the file "SWSV016-01 Weather Sealing_Ex/cohorts.csv", comment out rows for Htl (# in skip column)
2. Then, copy the file "SWSV016-01 Weather Sealing_Ex/cohorts.csv" to "SWSV016-01 Weather Sealing_Htl_Ex/cohorts.csv"
3. Then, in the new cohorts file ("SWSV016-01 Weather Sealing_Htl_Ex/cohorts.csv"), remove rows other than Htl.